.github/workflows/apply-hot-fixes.yml

```diff
@@ -0,0 +1,16 @@
+name: Apply hot fixes to master
+on:
+  push:
+    branches:
+      - release/v*.*.*
+jobs:
+  auto-pull-request:
+    name: PullRequestAction
+    runs-on: ubuntu-latest
```

Comment:
not an expert here but should we be using ubuntu-latest here? or should we stick to a version

Reply:
It is not possible to specify an exact [version for your os](https://github.com/actions/virtual-environments)

The difference between `ubuntu-latest` and `ubuntu-20.04` is at some point `ubuntu-latest` is going to change to the next version `ubuntu-22.04`.

I think this is fine, as all the most popular actions we are kept updated to date with those changes. We might have a broken action when `ubuntu-latest` changes, but this is a once a year thing.

Most importantly, the libraries/tools of `ubuntu-latest` are not used to build our code, our code is built inside a more controlled docker environment.